### PR TITLE
feat: unassign command

### LINKED ISSUE #207: `/unassign` command
This should allow a bounty hunter to release a bounty back into the DevPool (assign to @ubiquity-bounties)

              We need to actually create the `/unassign` function for this feature to work!

_Originally posted by @pavlovcik in https://github.com/ubiquity/bounty-bot/pull/196#pullrequestreview-1357989238_

### PR DESCRIPTION
**Pull request by Steveantor**
Resolves #207

### CODE REVIEW

File: src/handlers/comment/handlers/unassign.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return;
+  }
+
+  const issue_number = issue!.number;
+  const _assignees = payload.issue?.assignees;
+  const assignees = _assignees ?? [];
+  let shouldUnassign = false;
+  assignees.forEach((assignee) => {
+    if (assignee === payload.sender.login) {
+      shouldUnassign = !shouldUnassign && true;
```
Comment (pavlovcik) [inline src/handlers/comment/handlers/unassign.ts:29]: I'm confused about this `&& true` syntax

**Comment by pavlovcik** [comment]
This needs to "unassign" by removing all assignees, and then assigning @ubiquity-bounties for now. Please make sure to pull the name from the shared "reserved user names" array or whatever it is, instead of hardcoding in the function.

Also we will be deprecating that username soon, so however you implement this should be easy to catch to update as well ideally.

**Comment by Steveantor** [comment]
> This needs to "unassign" by removing all assignees, and then assigning @ubiquity-bounties for now. Please make sure to pull the name from the shared "reserved user names" array or whatever it is, instead of hardcoding in the function.
> 
> Also we will be deprecating that username soon, so however you implement this should be easy to catch to update as well ideally.

This is what it does,
- remove previous assignees
- Add reserved assignees `GLOBAL_STRING.assignees`

@pavlovcik

File: src/handlers/comment/handlers/unassign.ts (hunk 1 above)
Comment (Steveantor) [reply src/handlers/comment/handlers/unassign.ts:29]: it's basically shouldUnassign = true

Comment (0xcodercrane) [reply src/handlers/comment/handlers/unassign.ts:29]: We want 1 active assignee at a time so you can update it like 
`shouldUnassign = assignees.length > 0 && payload.sender.login === assignees[0]`

**Review by 0xcodercrane — CHANGES_REQUESTED** [review]

**Review by 0xcodercrane — APPROVED** [review]

**Pull request merged**